.circleci/publish_helm_charts.sh

```diff
@@ -34,7 +34,7 @@ helm repo index . --url $LOCAL_HELM_MOJALOOP_REPO_URI
 git status
 
 echo "Staging packaged Helm charts..." | tee git.log
-git add ./*.* ../README.md ../LICENSE.md ../CODEOWNERS
+git add ./*.tgz ./index.yaml ../README.md ../LICENSE.md ../CODEOWNERS
```

Comment:
If we are going to specific here, please also add support for providence files.

I.e. https://helm.sh/docs/topics/provenance/